com.unity.render-pipelines.universal/Runtime/ShaderUtils.cs

```diff
@@ -25,6 +27,8 @@ namespace UnityEngine.Rendering.Universal
             "Universal Render Pipeline/Simple Lit",
             "Universal Render Pipeline/Unlit",
             "Universal Render Pipeline/Terrain/Lit",
+            "Universal Render Pipeline/Nature/SpeedTree7",
+            "Universal Render Pipeline/Nature/SpeedTree7 Billboard",
```

Comment:
what about speedtree8?

Reply:
Ah yeah, I should probably add that as well. The test scene I was using only had SpeedTree7 shaders but I'll add an upgrader for Nature/SpeedTree8 -> "Universal Render Pipeline/Nature/SpeedTree8"

Reply:
Done in https://github.com/Unity-Technologies/Graphics/pull/1350/commits/cae0ba0b7a3c48d28f80f74322ba22707a9581c2